ruby/forwardfinancing.rb

```diff
@@ -13,60 +14,62 @@
   "lead" => {
     "contacts_attributes" => [
       {
-        "first_name" => "string",
-        "last_name" => "string",
-        "email" => "test@forwardfinancing.com",
-        "title" => "string",
-        "born_on" => "2015-01-01",
-        "home_phone" => "6176781000",
-        "cell_phone" => "6176781000",
-        "ssn" => "234345566",
-        "ownership_date" => "2015-01-01",
+        "first_name" => "#{Faker::Name.first_name}",
+        "last_name" => "#{Faker::Name.last_name}",
+        "email" => "#{Faker::Internet.email}",
+        "title" => "#{Faker::Name.prefix}",
+        "born_on" => "#{Faker::Date.birthday(18, 65).strftime("%Y-%m-%d")}",
+        "home_phone" => "#{Faker::Number.number(10)}",
+        "cell_phone" => "#{Faker::Number.number(10)}",
+        "ssn" => "#{Faker::IDNumber.valid}",
+        "ownership_date" => "#{Faker::Date.birthday(18, 65).strftime("%Y-%m-%d")}",
```

Comment:
Most of these examples of `Faker` use already return strings. You don't need to wrap a string, `s`, in `"#{s}"` since `s == "#{s}"`.

So, change things to `"ssn" => Faker::IDNumber.valid,`...

Reply:
Ah! Okay removed it